### ISSUE TITLE
This site can’t be reached

### ISSUE THREAD

**Issue by melissajenner22** [cloudposse/terraform-aws-elasticsearch#87]
I provisioned Elasticsearch. I got URL outputs of "domain_endpoint", "domain_hostname", "kibana_endpoint" and "kibana_hostname". But, I cannot hit any of these URLS. I got, "This site can’t be reached". Below is the code: 
```
main.tf:

module "elasticsearch" {
  source                  = "git::https://github.com/cloudposse/terraform-aws-elasticsearch.git?ref=tags/0.24.1"
  security_groups                = [data.terraform_remote_state.vpc.outputs.default_security_group_id]
  vpc_id                         = data.terraform_remote_state.vpc.outputs.vpc_id
  zone_awareness_enabled         = var.zone_awareness_enabled
  subnet_ids                     = slice(data.terraform_remote_state.vpc.outputs.private_subnets, 0, 2)
  elasticsearch_version          = var.elasticsearch_version
  instance_type                  = var.instance_type
  instance_count                 = var.instance_count
  encrypt_at_rest_enabled        = var.encrypt_at_rest_enabled
  dedicated_master_enabled       = var.dedicated_master_enabled
  create_iam_service_linked_role = var.create_iam_service_linked_role
  kibana_subdomain_name          = var.kibana_subdomain_name
  ebs_volume_size                = var.ebs_volume_size
  dns_zone_id                    = var.dns_zone_id
  kibana_hostname_enabled        = var.kibana_hostname_enabled
  domain_hostname_enabled        = var.domain_hostname_enabled
  allowed_cidr_blocks            = ["0.0.0.0/0"]
  advanced_options = {
    "rest.action.multi.allow_explicit_index" = "true"
  }
  context = module.this.context
}
```
terraform.tfvars:
```
enabled = true
region = "us-west-2"
namespace = "dev"
stage = "abcd"
name = "abcd"
instance_type = "m5.xlarge.elasticsearch"
elasticsearch_version = "7.7"
instance_count = 2
zone_awareness_enabled = true
encrypt_at_rest_enabled = false
dedicated_master_enabled = false
elasticsearch_subdomain_name = "abcd"
kibana_subdomain_name = "abcd"
ebs_volume_size = 250
create_iam_service_linked_role = false
dns_zone_id = "Z08006012JKHYUEROIPAD"
kibana_hostname_enabled = true
domain_hostname_enabled = true
```

**Comment by Gowiem** [comment]
Hey @melissajenner22, got your email about this. This is likely due to your launching your ES cluster into a VPC, which makes it not available to the public. You must be inside the network to access the cluster and it can be frustrating. 

I suggest checking out this documentation to get a better understanding of what you're running into: https://docs.aws.amazon.com/elasticsearch-service/latest/developerguide/es-vpc.html

Few ways you can solve this type of issue:
1. Port forward that address through a bastion host.
1. Use a VPN to access your internal resources like your ES cluster.
1. Use something like https://github.com/abutaha/aws-es-proxy to proxy connections into the cluster. 

It's a real PITA 😉  Please let me know if you have any further questions here, otherwise feel free to close this issue out!

**Comment by melissajenner22** [comment]
Hi @Gowiem, Thanks for your response. I did the following. And have a question.
1. Removed the line, allowed_cidr_blocks = ["0.0.0.0/0"] from the code.
2. Provisioned AWS Client VPN.
3. Login to AWS Client VPN.
4. Try to hit the page of kibana_endpoint. But, I got permission denied.
5. I logon to AWS console, navigate to ElasticSearch dashboard. For the Domain access policy, I chose Allow open access to the domain. See the policy below:
```
{
  "Version": "2012-10-17",
  "Statement": [
    {
      "Effect": "Allow",
      "Principal": {
        "AWS": "*"
      },
      "Action": "es:*",
      "Resource": "arn:aws:es:us-west-2:123456789098:domain/abcd-domain/*"
    }
  ]
}
```
I am able to hit the kibana_endpoint. My question is, how to add the above policy via Terraform code?
